lib/ipmi_backend_utils.pm

```diff
@@ -44,7 +44,7 @@ sub get_dom0_serialdev {
     my $dom0_serialdev;
 
     script_run("clear");
-    script_run("cat ${root_dir}/etc/SuSE-release");
+    script_run("cat ${root_dir}/etc/SuSE-release || cat ${root_dir}/etc/os-release");
```

Comment:
Nitpicking: Can't we use only `os-release` here? This file should exist on all SLE releases starting (and including) SLES12.

Reply:
Yeah, but not on SLE11SP4 which we still support partially in our tests.

Reply:
Yes, technically we can. But we need to cut more needles. So to save this effort, use this.